packages/ripple-nuxt-tide/modules/publication/pages/PrintAll.vue

```diff
@@ -105,7 +105,7 @@ export default {
     publishingInfo () {
       if (this.publication.type === 'node--publication') {
         const authorsName = this.publication.field_publication_authors ? this.publication.field_publication_authors : ''
-        const author = authorsName && authorsName.length > 0 ? authorsName.map((authors) => authors.name) : []
+        const author = authorsName ? authorsName.name : ''
```

Comment:
@yeniatencio I think we should support both an array and a single here so we dont break this behaviour anywhere?

Reply:
See https://github.com/dpc-sdp/ripple-framework/pull/837/files#diff-0b5c2b5d5780442faddb8f6ba0cba008efbdaa0710114f62735bde65f8db61a9R45

Reply:
@dylankelly , I have updated my pr for your review. Thank you.